src/components/capture/GenerateButton.tsx

```diff
@@ -9,25 +9,15 @@ import { useRouteStore } from 'hooks/useRouteStore';
 import { useZustandStore } from 'hooks/useZustand';
 import { isEmpty } from 'lodash';
 import { FormattedMessage } from 'react-intl';
-import {
-    entityCreateStoreSelectors,
-    formInProgress,
-    FormStatus,
-} from 'stores/Create';
+import { entityCreateStoreSelectors, FormStatus } from 'stores/Create';
 
 interface Props {
     disabled: boolean;
-    formId: string;
-    onFailure: Function;
+    callFailed: Function;
     subscription: Function;
 }
 
-function CaptureTestButton({
-    disabled,
-    formId,
-    onFailure,
-    subscription,
-}: Props) {
+function CaptureGenerateButton({ disabled, callFailed, subscription }: Props) {
```

Comment:
Test was previously used to generate stuff... so now that is the generate button